### ISSUE TITLE
I think there is some issue with block swap, am i right?

### ISSUE THREAD

**Issue by SamTyurenkov** [kohya-ss/musubi-tuner#502]
Before musubi-tuner i only used block swap with inference in comfyui, usually it was steadily loading RAM during entire generation.

When i set block swap with musubi-tuner, i don't see RAM loaded at all:

<img width="956" height="722" alt="Image" src="https://github.com/user-attachments/assets/80e63813-98cf-4e90-a492-a8742a87ae2a" />

Can this be environment specific issue? Or it's not designed for training at all? I was expecting to see RAM loaded during training, but it's not.

Currently running command is below:

`/workspace/venv_musubi/bin/accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 src/musubi_tuner/wan_train_network.py --mixed_precision bf16 --task t2v-14B --dit /workspace/models/diffusion_models/wan2.1_t2v_14B_bf16.safetensors --dataset_config /home/comfyuser/MusubiConfigs/config/lora_v8.toml --sdpa --optimizer_type AdamW8bit --learning_rate 0.0003 --lr_scheduler constant --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 8 --network_alpha 4 --network_dropout 0.1 --timestep_sampling shift --discrete_flow_shift 5.0 --max_train_epochs 16 --weighting_scheme logit_normal --num_timestep_buckets 2 --min_timestep 50 --max_timestep 900 --save_every_n_epochs 1 --seed 42 --scale_weight_norms 1 --output_dir /workspace/models/loras/training/wan --output_name lora_v8 --save_last_n_epochs 10 --max_grad_norm 1.0 --logit_mean 0.05 --logit_std 0.03 --sigmoid_scale 5 --preserve_distribution_shape --blocks_to_swap 30`

**Comment by kohya-ss** [comment]
There are many unknowns about how block swap is shown in the OS. It may be treated as a VRAM consumption rather than main memory consumption. If you decrease the value of `--block_to_swap` and an OOM occurs, it seems fine to assume that block swap is working correctly.

**Comment by SamTyurenkov** [comment]
> There are many unknowns about how block swap is shown in the OS. It may be treated as a VRAM consumption rather than main memory consumption. If you decrease the value of `--block_to_swap` and an OOM occurs, it seems fine to assume that block swap is working correctly.

I think i had that. Ok
